README.md

```diff
@@ -1,10 +1,8 @@
-# EJS
-
-Embedded JavaScript templates
-
+Embedded JavaScript templates<br/>
```

Comment:
What does this do? I'm not necessarily keen on adding raw HTML markup to an MD file.

Reply:
It&nbsp;ensures that&nbsp;a&nbsp;line&nbsp;break is&nbsp;added between&nbsp;the&nbsp;main&nbsp;header and&nbsp;the&nbsp;badges.

The&nbsp;alternative would&nbsp;be&nbsp;to&nbsp;add two&nbsp;normal&nbsp;spaces at&nbsp;the&nbsp;end of&nbsp;the&nbsp;line, which&nbsp;is&nbsp;the&nbsp;markdown&nbsp;syntax for&nbsp;`<br/>`, but&nbsp;that&nbsp;runs the&nbsp;risk of&nbsp;getting automatically&nbsp;removed by&nbsp;editors as&nbsp;part of&nbsp;“**Trim&nbsp;Trailing&nbsp;Whitespace**”.

Reply:
👍🏻